<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>19</y>
     <width>691</width>
     <height>341</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Control_Config">
    <attribute name="title">
     <string>Controle</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QPushButton" name="MC_Control_Button">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>00</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="MR_Control_ComboBox">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <item>
           <property name="text">
            <string>Mover para frente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para tras</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para direita</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para esquerda</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="MC_Control_ComboBox">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <item>
           <property name="text">
            <string>Mover para cima</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para baixo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para direita</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para esquerda</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="MR_Control_Button">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>00</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QComboBox" name="Others_Control_ComboBox">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <item>
           <property name="text">
            <string>Aumentar a velocidade</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Diminuir a velocidde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gravar</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="Others_Control_Button">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="Control_ButtonBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Typesetter_Config">
    <attribute name="title">
     <string>Teclado</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="7" column="0">
         <widget class="QComboBox" name="Others_TypeSetter_ComboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <item>
           <property name="text">
            <string>Aumentar a velocidade</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Diminuir a velocidde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gravar</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="Others_TypeSetter_Button">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="MR_TypeSetter_ComboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Mover para frente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para tras</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para direita</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para esquerda</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="MC_TypeSetter_Button">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QComboBox" name="MC_TypeSetter_ComboBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <item>
           <property name="text">
            <string>Mover para cima</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para baixo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para direita</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mover para esquerda</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="MR_TypeSetter_Button">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="TypeSetter_ButtonBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
